src/style/styles.ts

```diff
@@ -0,0 +1,5 @@
+export const productImageStyles = {
```

Comment:
type this object as React.CSSProperties so that you don't need to typecast it where you are using it